openapi_python_client/templates/model.pyi

```diff
@@ -25,37 +26,37 @@ class {{ model.reference.class_name }}:
     """ {{ model.description }} """
     {% for property in model.required_properties + model.optional_properties %}
     {% if property.default is none and property.required %}
-    {{ property.to_string() }}
+    _{{ property.to_string() }}
```

Comment:
The prepending of _ to properties is the only thing not currently doable at the head/tail of the file.